src/resources/OrganizationAccess/AccessInterfaces.ts

```diff
@@ -21,10 +21,58 @@ export interface AccessParams {
     privilegeTargetDomain: string;
 }
 
-export interface AccessModel {
-    accessLevel: AccessLevel[];
+export type GroupAccessModel = {
+    /*
+     * The access level granted by the access model.
+     */
+    accessLevel: AccessLevel;
+    /*
+     * Whether the calling user is part of this group.
+     */
     callerPartOf: boolean;
+    /*
+     * The display name of the group.
+     */
     displayName: string;
+    /*
+     * The id of the group.
+     */
     id: string;
-    privilegeholderType: PrivilegeholderType;
-}
+    /*
+     * Represent the group access model type.
+     */
+    privilegeHolderType: PrivilegeHolderType.GROUP;
+    /*
+     * The list of resources ids this access model has edit access level on.
```

Comment:
Now that I think about it, after the interface split we can even get more precise here. Feels clearer IMO

```suggestion
     * The list of resources ids this group has edit access level on.
```